dfetch/project/vcs.py

```diff
@@ -101,6 +102,10 @@ def update(self, force: bool = False) -> None:
 
         actually_fetched = self._fetch_impl(to_fetch)
         self._log_project(f"Fetched {actually_fetched}")
+
+        if self.__project.patch and os.path.exists(self.__project.patch):
```

Comment:
Could you add this to the uml diagram in `docs/static/uml`?

Reply:
Done